keyboards/keebit/coral/config.h

```diff
@@ -0,0 +1,10 @@
+// Copyright 2023 numToStr (@numToStr)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 220U
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP16
+
+#define WS2812_PIO_USE_PIO1
```

Comment:
Is there a reason for this line?

Reply:
When I was building the firmware, docs mentioned something and I thought it was somewhat necessary. I'll remove it now.

Reply:
It's only necessary when there is a PIO conflict (for example with the split serial driver) - if everything works fine without it, you don't need it. However as this is a split board and you are using the `vendor` PIO driver (which also defaults to PIO0), it's likely required to move the WS2812 driver to the other PIO peripheral.

Reply:
Got it. Thanks for the explanation. I'll leave it as it is then :)